common/lib/xmodule/xmodule/modulestore/tests/test_cross_modulestore_import_export.py

```diff
@@ -448,3 +452,73 @@ def test_round_trip(self, source_builder, dest_builder, source_content_builder,
                             dest_store,
                             dest_course_key,
                         )
+
+    @ddt.data(*itertools.product(
+        [SPLIT_MODULESTORE_SETUP],
+        [SPLIT_MODULESTORE_SETUP],
+        CONTENTSTORE_SETUPS,
+        CONTENTSTORE_SETUPS,
+        ['split_course_with_static_tabs'],
+    ))
```

Comment:
Sorry I didn't notice this before - now that you're testing a single modulestore type vs. a single contentstore type vs. a single course, there's no need for the DDT here. What you have here will work - but it's more complex than it needs to be. You should simply call out the specific modulestore/contentstore/course below during construction.